app/src/main/java/org/mozilla/focus/activity/InfoActivity.java

```diff
@@ -0,0 +1,69 @@
+/* -*- Mode: Java; c-basic-offset: 4; tab-width: 4; indent-tabs-mode: nil; -*-
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.activity;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+
+import org.mozilla.focus.R;
+
+/**
+ * A generic activity that supports showing additional information in a WebView. This is useful
+ * for showing any web based content, including About/Help/Rights, and also SUMO pages.
+ */
+public class InfoActivity extends AppCompatActivity {
+    private static final String EXTRA_URL = "extra_url";
+    private static final String EXTRA_TITLE = "extra_title";
+
+    public static final Intent getIntentFor(final Context context, final String url, final String title) {
+        final Intent intent = new Intent(context, InfoActivity.class);
+
+        intent.putExtra(EXTRA_URL, url);
+        intent.putExtra(EXTRA_TITLE, title);
+
+        return intent;
+    }
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        setContentView(R.layout.activity_info);
+
+        final String url = getIntent().getStringExtra(EXTRA_URL);
+        final String title = getIntent().getStringExtra(EXTRA_TITLE);
+
+        final WebView webView = (WebView) findViewById(R.id.webview);
+        // By default WebView will try to open links and redirects in an external browser (and sumo
+        // links usually use a redirect to get to the desired target page). Somehow, setting a default
+        // WebViewClient() is enough to disable that behaviour.
+        webView.setWebViewClient(new WebViewClient());
```

Comment:
As far as I can tell, there's only one webview cache per (Android) application, which all WebViews then reuse. I don't actually see any way of overriding this behaviour - which means we might have to share data between BrowserFragment and here. Clearing the cache/history/etc will then affect an existing BrowserFragment, which we probably don't want either.

But: as far as I can tell: history is specific to a WebView (so we don't need to care about that), actually shared data is restricted to the cache (which we can clear without huge harm to the current session, except for increased data usage), cookies (we could remove/restore sumo cookies), WebStorage (same as cookies), maybe more?